pages/linux/ntpq.md

```diff
@@ -0,0 +1,23 @@
+# ntpq
+
+> Query Network Time Protocol (NTP) daemon.
+
+- Use ntpq in interactive mode:
+
+`ntpq -i`
```

Comment:
```suggestion
`ntpq --interactive`
```
For reference: the long options are neither listed in the official man page, nor by `ntpq --help`, but they are in <https://manned.org/ntpq> and they do work